product-service/src/handlers/getProductsList.ts

```diff
@@ -1,45 +1,47 @@
 import { APIGatewayProxyHandler, APIGatewayProxyResult } from 'aws-lambda';
 import 'source-map-support/register';
-import productList from '../productList.json';
-import type { ProductList } from '../models/product';
-import getAgeByName from '../services/agify/methods/getAgeByName';
+import { Client } from 'pg';
+import dbOptions from '../dbConfig';
+
+console.log('DB CONFIG::', dbOptions);
+
+const getAllProductsQuery = `
+select 
+  p.id, p.title, p.description, p.price, s.count 
+from 
+  products p 
+left join stocks s 
+ ON id = product_id
+`;
+
+const defaultHeaders = {
+  'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+  'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+};
 
 export const handler: APIGatewayProxyHandler = async (): Promise<
   APIGatewayProxyResult
 > => {
+  console.log('getProductList handler, no arguments');
```

Comment:
You could log event object, because FE can pass some useful headers (like session-id for example) and later on you could filter logs by these headers if logging system supports it of course

Reply:
Indeed, but in task there were stated that we should log arguments, that's why it's only arguments for handler here